doc/tutorials/content/gpu_octree.rst

```diff
@@ -58,19 +58,21 @@ Note that unlike in the CPU octree implementation, the user cannot set the resol
 
 .. literalinclude:: sources/gpu_octree/gpu_octree.cpp
     :language: cpp
-    :lines: 47-58
+    :lines: 107-118
 
-After building the octree, we we can move on to the different search queries.
+After building the octree, we can move on to the different search queries.
 
 K Nearest Neighbors search
 ---------------------------
 
-The K nearest neighbor algorithm is designed to identify a given (k) number of nearest neighbors to a query point. However, the GPU KNN search algorithm is currently restricted to k=1 only.
 
+.. note::
+
+    The K nearest neighbor algorithm is designed to identify a given (k) number of nearest neighbors to a query point. However, the GPU KNN search algorithm is currently restricted to k=1 only.
 
 .. note::
 
-    The GPU K nearest neighbor algorithm differs from it's CPU counterpart in implementation. Specifically, the CPU algorithm will only find approximate nearest neighbors, where the user can set the upper bound of the error allowed. In contrast, the GPU algorithm finds the absolute nearest neighbor.
+    The GPU K nearest neighbor algorithm differs from it's CPU counterpart in implementation. Specifically, the CPU algorithm will only find approximate nearest neighbors, where the user can set the upper bound of the error allowed. In contrast, the GPU algorithm finds the absolute nearest neighbor. Unless it is necessary to find the absolute nearest neighbor, users can utilize approximate nearest search instead, as both functions offer similar functionality until support is added for K>1.
```

Comment:
The last sentence you is mildly confsing. Focus on establishing the use conditions:

* if the user only cares about K=1 and an approximation is ok -> use gpu approx nn